config/routes/api/v2.rb

```diff
@@ -245,6 +245,7 @@
           resources :table_preferences, :only => [:index, :create, :destroy, :show, :update]
           resources :mail_notifications, :only => [:create, :destroy, :update]
           get 'mail_notifications', :to => 'mail_notifications#user_mail_notifications', :on => :member
+          delete :invalidate_jwts, :to => :invalidate_jwts, :on => :member
```

Comment:
Should this be a subcontroller for managing JWTs? I can imagine desire for other endpoints - creating tokens, listing active tokens etc.

Reply:
We don't store tokens in our DB, the only endpoint I can think of is the creation. That would potentially make sense then. But it would feel as an overkill for just resetting the secret base to invalidate all the signatures. (not a strong feeling)